README.md

```diff
@@ -18,93 +16,681 @@
+$ cd getml-demo/docker  
+$ docker-compose up getml_on_3_8  
+```
+
+The files are set up to also work with [podman](https://podman.io/) and [podman-compose](https://github.com/containers/podman-compose)
+
+```
+$ podman-compose up getml_on_3_11
+```
+
+> [!IMPORTANT]  
+> As the services are set up to use the same ports and volumes, you can't run both at the same time. 
+
+To open Jupyter Lab in the browser, look for the following lines in the output and copy-paste it in your browser:
+
+```
+Or copy and paste one of these URLs:
+
+http://localhost:8888/lab?token=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+```
 
+After the first `getml.engine.launch(...)` is executed and the engine is started, its monitor can be opened in the browser under
 
-## Try it live
+```
+http://localhost:1709/#/token/token
+```
 
-You can run these notebooks in a jupyterhub environment on our server. No installation or registration required. Start your own getML live demo [here](https://demo.getml.com/v2/gh/getml/getml-demo/master?urlpath=lab). (Launching your docker container might take a minute or two, please be patient.)
+### On the Machine (Linux only)
 
-You can also download getML for free [here](https://www.getml.com/product/) to run these notebooks locally on your machine.
+Alternatively, getML and the notebooks can be run natively on the local Linux machine by having certain software installed, like Python and some Python libraries, Jupyter-Lab and the getML engine. The [getML Python library](https://github.com/getml/getml-community/) provides an engine version without [enterprise features](https://www.getml.com/pricing). But as those features are shown in the demonstration notebooks, the [trail of the enterprise version](https://www.getml.com/download) can be used for those cases.
 
-## Examples
+The following commands will set up a Python environment with necessary Python libraries and the trail of the getML enterprise version, and Jupyter-Lab
 
-This demo contains various example projects to help you to get started
-with relational learning and getML. They cover different aspects of the software, and can serve as documentation or
-as blueprints for your own project.
+```
+$ git clone https://github.com/getml/getml-demo.git  
+$ cd getml-demo  
+$ pipx install hatch
+$ hatch env create
+$ hatch shell
+$ pip install -r requirements/requirements.3.11.txt
+$ jupyter-lab
+```
+
+> [!TIP]  
+> Install the [trail of the enterprise version](https://www.getml.com/download) via the  [Install getML on Linux guide](https://docs.getml.com/latest/home/installation/linux.html#install-getml-on-linux) to try the enterprise features.
+
+With the last command, Jupyter-Lab should automatically open in the browser. If not, look for the following lines in the output and copy-paste it in your browser:
+
+```
+Or copy and paste one of these URLs:
+
+http://localhost:8888/lab?token=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
Please find a better placeholder for `xxxxxx...`. Candidate: `<randomly_generated_token>`